arquillian-recorder-reporter/arquillian-recorder-reporter-api/src/main/java/org/arquillian/recorder/reporter/model/entry/FileEntry.java

```diff
@@ -86,4 +86,29 @@ public String getMessage() {
     public void setMessage(String message) {
         this.message = message;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
Yes @lordofthejars.

@bartoszmajsak , instanceof check returns true for subclasses too, so its not strictly equals comparison